eland/query.py

```diff
@@ -195,8 +195,8 @@ def composite_agg_start(
 
         Parameters
         ----------
-        size: int
-            Pagination size.
+        size: int or None
+            Use composite aggregation with pagination if size is Not None
```

Comment:
nit: lowercase `not`, also when is `size=None`?

Reply:
- size=None is for checking, If we want pagination or not.
- If size is given, pagination can be performed.

Reply:
Ok, Sorry size=None doesn't make sense